tests/console/container_base_images.pm

```diff
@@ -41,12 +50,14 @@ sub run {
         clean_docker_host();
     }
     # Run podman tests
-    if (is_sle || check_var("SKIP_PODMAN_IMAGE_TESTS", 1)) {
+    if (skip_podman) {
         record_info("Skip Podman", "Podman image tests skipped");
         script_run("echo 'INFO: Podman image tests skipped' >> /var/tmp/container_base_images_log.txt");
     } else {
+        # In SLE we need to add the Containers module
+        add_suseconnect_product(get_addon_fullname('contm'));
         zypper_call('in podman podman-cni-config', timeout => 900);
-        foreach my $image (@images, @podman_images) {
+        foreach my $image (@podman_images) {
             test_container_image($image, 'latest', 'podman');
             script_run("echo 'OK: podman - $image:latest' >> /var/tmp/container_base_images_log.txt");
         }
```

Comment:
Personally I would go for hashes of variables based on "docker" and "podman" in order to avoid having those lines duplicated, and probably only branch between
```
        add_suseconnect_product(get_addon_fullname('contm'));
        zypper_call('in podman podman-cni-config', timeout => 900);
```
and 
`install_docker_when_needed();`

In that case of course `skip_docker()` and `skip_podman()` would only be one function with `docker` or `podman` as argument.

However this looks good and I think that both ways have their merits.

Reply:
I played around with removing the code redundancies and moved the for loop into a new subroutine.

Reply:
IMHO the hashes approach makes the test more complicated without a real benefit at the moment, but could be something to keep in mind, when the test grows